R/cml02a_gl.R

```diff
@@ -0,0 +1,50 @@
+# cml02a_gl_main ----
+
+#' @describeIn cml02a_gl Main TLG function
+#'
+#' @inheritParams gen_args
+#' @returns the main function returns an `rlistings` or a `list` object.
+#' @export
+#'
+cml02a_gl_main <- modify_default_args(std_listing,
+  dataset = "adcm",
+  key_cols = c("ATC2", "CMDECOD"),
+  disp_cols = c("ATC2", "CMDECOD", "CMTRT"),
+  split_into_pages_by_var = NULL
+)
+
+#' @describeIn cml02a_gl Preprocessing
+#'
+#' @inheritParams cml02a_gl_main
+#' @inheritParams gen_args
+#' @returns the preprocessing function returns a `list` of `data.frame`.
+#'
+#' @export
+#'
+cml02a_gl_pre <- function(adam_db,
```

Comment:
why pre functions and main functions have different default values (when using `run` they should have the same arguments)